tests/e2e/specs/productDetailPage_spec.js

```diff
@@ -0,0 +1,49 @@
+describe('Product detail page', () => {
+  before(() => {
+    cy.visit('/product/lemare-booties-0778-grey/M0E20000000E0WX');
+  });
+
+  it('Displays a single product details', () => {
+    cy.get('[data-test=pdp-product-title]', { timeout: 20000 })
+      .contains('Booties Lemare grey')
+      .parentsUntil('[data-test=product-description]')
+      .parent()
+      .find('[data-test=quickview-sku]')
+      .contains('M0E20000000E0WX')
+      .parentsUntil('[data-test=product-description]')
+      .parent()
+      .then(($product) => {
+        cy.wrap($product)
+          .find('[data-test=product-discounted-price]')
+          .should('contain', '248,75 €');
+        cy.wrap($product)
+          .find('[data-test=product-discount-price]')
+          .should('contain', '174,12 €');
+
+        cy.wrap($product)
+          .find('[data-test=add-to-cart-button]')
+          .should('exist');
```

Comment:
This, as it is, it's a bit unnecessary. We will better test this button on the add to cart e2e.